app/templates/views/guidance/using-notify/attach-pages.html

```diff
@@ -0,0 +1,45 @@
+{% extends "content_template.html" %}
+
+{% from "components/service-link.html" import service_link %}
+
+{# Used by the content_template.html layout, prefixes the "navigation" accessible name #}
+{% set navigation_label_prefix = 'Using Notify' %}
+
+{% block per_page_title %}
+  Attach pages to a letter template
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Attach pages to a letter template</h1>
+
+    <p class="govuk-body">Upload a PDF and we’ll print it as part of your letter.</p>
+    <p class="govuk-body">You can attach pages with complex layouts. For example, forms, tables or direct debit mandates.</p>
```

Comment:
```suggestion
    <p class="govuk-body">You can attach pages with complex layouts. For example, forms, tables or Direct Debit mandates.</p>
```

I think `Direct Debit` is usually capitalised: https://www.directdebit.co.uk/direct-debit-explained/what-is-direct-debit/